### PR TITLE
Update How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino…

### PR DESCRIPTION
**Pull request by ben-jacobson**
…-UNO.md

There was a missing step in these instructions. Also, I can't change this in Github but the image has an error in it, it shows a connection to ATMega IOREF rather than RST. It's one pin over.

**Changes proposed in this pull request**
- Example 1
- Example 2

**Please check**
- [ ] This pull request changes an article title
- [ ] This pull request removes an article

### CODE REVIEW

File: content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md [hunk 1]
```diff
@@ -44,11 +44,13 @@ The bootloader is a piece of code stored in a reserved space in your Arduino boa
 
 08. Connect the Arduino Uno to your PC again.
```
Comment (seaxwi) [inline content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:45]:
```suggestion
08.  Connect the Arduino Uno to your PC again and make sure it's selected in `Tools > Port
```

File: content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md [hunk 2]
```diff
@@ -44,11 +44,13 @@ The bootloader is a piece of code stored in a reserved space in your Arduino boa
...
 
-09. Check the board and port selection like in step 2 and 3.
+09. Check the port selection like in step 2 and 3.
```
Comment (seaxwi) [inline content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:47]:
```suggestion
09. Select `Tools > Board > Arduino AVR Boards > Arduino Mega or Mega 2560`.
```

File: content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md [hunk 3]
```diff
@@ -44,11 +44,13 @@ The bootloader is a piece of code stored in a reserved space in your Arduino boa
...
 
-10. Select `Tools > Programmer > Arduino as ISP`.
+10. Change the board to Arduino Mega or Mega 2560.
```
Comment (seaxwi) [inline content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:49]:
```suggestion
10. Select `Tools > Programmer > Arduino as ISP`.
```

File: content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md [hunk 4]
```diff
@@ -44,11 +44,13 @@ The bootloader is a piece of code stored in a reserved space in your Arduino boa
...
 
-11. Select `Tools > Burn Bootloader`.
+11. Select `Tools > Programmer > Arduino as ISP`.
+
+12. Select `Tools > Burn Bootloader`.
```
Comment (seaxwi) [inline content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:53]:
```suggestion
11. Select `Tools > Burn Bootloader`.
```

**Review by seaxwi — CHANGES_REQUESTED** [review]
Hi @ben-jacobson, thanks for contributing. Could you please sign our [license agreement](https://github.com/arduino/help-center-content/pull/47#issuecomment-1100800382)?

I've fixed the image you mentioned. You're right about the missing step. I don't think there's any point referring back to previous steps now that it's added. So how about this?

```
08. Connect the Arduino Uno to your PC again and make sure it's selected in `Tools > Port`.

09. Select `Tools > Board > Arduino AVR Boards > Arduino Mega or Mega 2560`.

10. Select `Tools > Programmer > Arduino as ISP`.

11. Select `Tools > Burn Bootloader`.
```

File: content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md (hunk 1 above)
Comment (per1234) [reply content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:45]: @seaxwi you are missing the closing backtick at the end of `` `Tools > Port` `` in your suggestion.

Comment (seaxwi) [reply content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md:45]: @per1234 fixed

**Comment by seaxwi** [comment]
Closing this since the issues are being covered in https://github.com/arduino/help-center-content/pull/63.

**Pull request closed without merging**